script.js

```diff
@@ -30,3 +30,145 @@ contactLink.addEventListener('click', () => {
+    featuredImage: 'images/snapashop-portfolio.png',
+    technologies: ['Ruby on rails', 'css', 'Javascript', 'html'],
+    liveVersion: '',
+    linkSource: '',
+  },
+];
+
+const cardsPortfolio = document.querySelector('.cards-portfolio');
+let portfolioSection = '';
+portfolioObj.forEach((item, index) => {
+  let technologies = '';
+  item.technologies.forEach((tech) => {
+    technologies += `<li>
+    <a href='#'
+      ><button class='btn-hover btn-active btn-pressed'>
+        ${tech}
+      </button></a
+    >
+  </li>`;
+  });
+
+  const newItem = `
+  <div class='project'>
+    <div class='image-back'></div>
+    <div class='project-footer'>
+      <div class='project-title-back'>
+        <h4>${item.name}</h4>
+      </div>
+      <ul class='project-buttons-back'>
+        ${technologies}
+      </ul>
+      <button id='${index}' class='works-button btn-hover btn-active btn-pressed'>
+        See Project
+      </button>
+    </div>
+  </div>
+  `;
+  portfolioSection += newItem;
+});
+
+cardsPortfolio.innerHTML += portfolioSection;
+
+const worksButtons = document.querySelectorAll('.works-button');
+const popupContainer = document.querySelector('.popup-container');
+const popupMenu = document.querySelector('.popup-menu');
+
+worksButtons.forEach((button) => {
+  button.addEventListener('click', (e) => {
+    const portfolioItem = portfolioObj[Number(e.target.id)];
+
+    let technologies = '';
+    portfolioItem.technologies.forEach((tech) => {
+      technologies += `<li>
+      <button class='btn-hover btn-active btn-pressed'>
+        ${tech}
+      </button>
+    </li>`;
+    });
+
+    popupMenu.innerHTML = `
+      <i class='fa fa-close' id='close-popup'></i>
+      <img alt='' src='${portfolioItem.featuredImage}' />
+      <h2>${portfolioItem.name}</h2>      
+      <div class='popup-buttons-desktop'>
+        <button>
+          See Live
+          <i class='fa-solid fa-arrow-up-right-from-square'></i>
+        </button>
+        <button>
+          See Source
+          <i class='fa-brands fa-github'></i>
+        </button>
+      </div>
+      <ul>
+        ${technologies}
+      </ul>
+      <p>
+        ${portfolioItem.description}
+      </p>
+      <div class='popup-buttons'>
+        <button>
+          Live
+          <i class='fa-solid fa-arrow-up-right-from-square'></i>
+        </button>
+        <button>
+          Source
+          <i class='fa-brands fa-github'></i>
+        </button>
+      </div>
+    `;
+    popupContainer.style.display = 'flex';
+    const closePopup = document.querySelector('#close-popup');
+    closePopup.addEventListener('click', () => {
+      popupContainer.style.display = 'none';
+    });
+  });
+});
```

Comment:
Nice work putting in some effort in making the changes requested by the previous reviewer😉 but your popup menu still looks different from the design😉 don't forget that it is a requirement that you stick to the design in this milestone🤗 kindly see the images below and make the necessary adjustments🤗

your popup
<img width="952" alt="your__popup desktop" src="https://user-images.githubusercontent.com/64643995/216030301-957a093b-4b00-45e1-abde-3a2964b6997b.png">

design
<img width="370" alt="pop_up design" src="https://user-images.githubusercontent.com/64643995/216030409-0d6820ab-89cc-467a-a545-f6a5c2dfff4a.png">

kindly take note of the spacings, font-size, font-family, button appearance and interactions (hover, focused etc) and make the necessary changes🤗🤗